test/land/test_imsproc.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -e
+################################################
+YY=2021
+MM=03
+DD=23
+HH=18
+FILEDATE=$YY$MM$DD.${HH}0000
+RES=48
+
+project_binary_dir=$1
+project_source_dir=$2
+
+GYMD=$(date +%Y%m%d -d "$YY$MM$DD $HH - 6 hours")
+GHR=$(date +%H -d "$YY$MM$DD $HH - 6 hours")
+DOY=$(date +%j -d "$YY$MM$DD + 1 day")
+
+EXECDIR=$project_source_dir/build/bin/
+WORKDIR=$project_binary_dir/test/testrun
+RSTDIR=$GDASAPP_TESTDATA/lowres/gdas.$GYMD/$GHR/atmos/RESTART/
+
+export OBSDIR=/scratch2/NCEPDEV/land/data/DA/snow_ice_cover
```

Comment:
we need to get the obs staged in the same testdata directory, let me do that ASAP on Hera and then later on Orion

Reply:
Try now `$GDASAPP_TESTDATA/land/snow_ice_cover/IMS/index_files` and `$GDASAPP_TESTDATA/land/snow_ice_cover/IMS/2021`

Reply:
Made the changes to use the same testdata directory. Thanks.